omero/sysadmins/python3-migration.rst

```diff
@@ -0,0 +1,128 @@
+so neither is particularly well supported at the moment. We are examining workarounds like
+building Ice from source and/or building a .deb package.
+
+Other prerequisites
+-------------------
+
+OMERO's other prerequisites have not changed substantially but if you would like to take this
+opportunity to move to the :doc:`recommended version<version-requirements>` for all requirements,
+the current choices are:
+
+- Ice 3.6 (non-optional)
+- Java 11
+- Nginx 1.14 or higher
+- PostgreSQL 11
+
+Other options
+-------------
+
+The installation walkthroughs provided in the documentation try to stick to a minimum installation.
+The only requirements are an understanding of the Unix shell, the standard package manager for your
+platform, and the regular Python distribution mechanisms.
+
+However, more advanced installation mechanisms are available if you are interested and have familiarity
+with the given mechanism:
+
+- `Ansible roles <https://galaxy.ansible.com/ome>`_ are available for most installation steps. The primary
+  roles, `omero-server` and `omero-web` have not yet been released and will need to be installed from
+  GitHub.
+
+- A `conda channel <https://anaconda.org/ome>_` provides pre-built packages needed by OMERO if you prefer
+  to use Anaconda/Miniconda instead of the Python distribution provided by your platform.
+
+- `Docker images <https://hub.docker.com/u/openmicroscopy>`_ are also available. Both the `omero-server`
+  and `omero-web` images are considered production quality.
+
+Please get in touch at https://forum.image.sc/c/data if you have any questions.
+
+OMERO.server
+------------
+
+The steps for an OMERO.server installation have not changed substantially.
+
+Download the OMERO.server.zip as you would usually do, and unpack it under your
+installation directory. We suggest: file:`/opt/omero/server/OMERO.server`
+
+We highly recommend a virtualenv-based installation for all of the Python
+dependencies. Follow the :doc:`standard installation instructions <unix/server-installation>`
+for your platform.
+All instructions ...
+
+Once you have your installation in place, you will need to follow the
+:doc:`standard upgrade instructions <server-upgrade>`, working from
+a *copy* of your data.
+
+OMERO.web
+---------
+
+Although it is possible to also follow the previous installation steps
+for OMERO.web, installation no longer requires downloading a package from
+https://downloads.openmicroscopy.org. If you choose to follow this route,
+all requirements will be installed directly into the virtualenv for OMERO.web.
+Instructions are available under :doc:`web-deployment <unix/install-web/web-deployment>`.
+
+The primary change is that it is now required to set the :envvar:`OMERODIR` variable
+to specify where the OMERO installation lives. This defines where configuration
+files and log files will be stored. Again, we suggest: file:`/opt/omero/web` as the
```

Comment:
``Again`` ? Maybe leave out, definitely this was not suggested in this text.

The sentence starting with ``Again`` - the path there could be better formatted, see screenshot below

![Screen Shot 2019-12-04 at 11 52 44](https://user-images.githubusercontent.com/2478303/70140513-c71cb000-168c-11ea-8d10-399a787fa4a8.png)